analysis/organize_data.qmd

```diff
@@ -47,188 +47,197 @@ gss <- gss |>
+table(gss$racecen2, gss$race, exclude=NULL)
+table(gss$hispanic, gss$race, exclude=NULL)
 ```
 
 ```{r}
-#| label: Recoding religious preference variable
+#| label: code-educ
 gss <- gss |>
-  mutate(religpref=case_when(
-    relig==1 ~ "pretestant",
-    relig==2 ~ "catholic",
-    relig==3 ~ "jewish",
-    relig==4 ~ "none",
-    relig==5 ~ "other",
-    relig==6 ~ "buddhism",
-    relig==7 ~ "hinduism",
-    relig==8 ~ "other eastern religions",
-    relig==9 ~ "muslim/islam",
-    relig==10 ~ "orthodox christian",
-    relig==11 ~ "chistrian",
-    relig==12 ~ "indigenous",
-    relig==13 ~ "non-denominational",
-    TRUE ~ NA_character_
-  ))
-
-table(gss$relig, gss$religpref, exclude=NULL)
+  mutate(educ_attain=factor(
+    case_when(
+      educ<=11 ~ "less than high school",
+      educ==12 ~ "high school",
+      educ>=13 & educ<=15 ~ "some college",
+      educ==16 ~ "bachelors",
+      educ>=17 ~ "graduate",
+      TRUE ~ NA_character_),
+    levels=c("less than high school","high school","some college","bachelors",
+             "graduate")))
+  
+table(gss$educ, gss$educ_attain, exclude=NULL)
 ```
 
 ```{r}
-#| label: Collapsing religious attendance
+#| label: code-relig-affil
 gss <- gss |>
-  mutate(religattend=case_when(
-    attend==0 ~ "never",
-    attend>=1 & attend<=2 ~ "yearly",
-    attend>=3 & attend<=4 ~ "monthly",
-    attend>=5 & attend<=7 ~ "weekly",
-    attend==8 ~ "more than weekly",
-    TRUE ~ NA_character_
-  ))
-
-table(gss$attend, gss$religattend, exclude=NULL)
-```
-
-```{r}
-#| label: Collapsing Prayer Frequency
+  mutate(relig_affil=factor(
+    case_when(
+      relig==1 ~ "protestant",
+      relig==2 ~ "catholic",
+      relig==10 | relig==11 | relig==13 ~ "other christian",
+      relig==3 ~ "jewish",
+      relig==4 ~ "none",
+      relig==6 ~ "buddhist",
+      relig==7 ~ "hindu",
+      relig==9 ~ "muslim",
+      is.na(relig) ~ NA_character_,
+      TRUE ~ "other"),
+    levels=c("protestant","catholic","other christian","jewish","muslim",
+             "buddhist","hindu","other","none")))
+
+table(gss$relig, gss$relig_affil, exclude=NULL)
+```
+
+```{r}
+#| label: code-service-attend
 gss <- gss |>
-  mutate(prayfreq=case_when(
-    pray==6 ~ "never",
-    pray>=4 & pray<=5 ~ "weekly",
-    pray>=2 & pray<=3 ~ "daily",
-    pray==1 ~ "more than daily",
-    TRUE ~ NA_character_
-  ))
+  mutate(relig_attend=factor(
+    case_when(
+      attend==0 ~ "never",
+      attend>=1 & attend<=2 ~ "yearly",
+      attend>=3 & attend<=4 ~ "monthly",
+      attend>=5 & attend<=7 ~ "weekly",
+      attend==8 ~ "more than weekly",
+      TRUE ~ NA_character_),
+    levels=c("never","yearly","monthly","weekly","more than weekly")))
 
-table(gss$pray, gss$prayfreq, exclude=NULL)
+table(gss$attend, gss$relig_attend, exclude=NULL)
 ```
 
 ```{r}
-#| label: Collapsing belief in god
+#| label: code-prayer-freq
 gss <- gss |>
-  mutate(bgod=case_when(
-    god>=2 ~ "no belief",
```

Comment:
This should have been `<=`. The check yourself table revealed that the code wasn't working  - check yourself tables don't work if you don't read them! 

I fixed it below.